server/src/main/java/io/deephaven/server/table/ops/RangeJoinGrpcImpl.java

```diff
@@ -82,7 +108,12 @@ public Table create(RangeJoinTablesRequest request, List<ExportObject<Table>> so
         final Table leftTable = sourceTables.get(0).get();
         final Table rightTable = sourceTables.get(1).get();
         final Collection<JoinMatch> exactMatches = JoinMatch.from(request.getExactMatchColumnsList());
-        final RangeJoinMatch rangeMatch = parseRangeMatch(request);
+        final RangeJoinMatch rangeMatch;
+        if (request.getRangeMatch().isEmpty()) {
```

Comment:
Regardless of whether we check empty or presence bit, I think we should have a helper function for the few callsites that are doing this.